### PR TITLE
Add more tests for "resume" operation

### PR DESCRIPTION
**Pull request by ivan-ochc**
Add more tests for "resume" operation; refactor existing tests.

### CODE REVIEW

**Comment by aleksuss** [comment]
@ivan-ochc I'd propose to use `black` utility for the code formatting to achieve the same code formatting style.

File: test-suite/exonum-py-tests/suite/common.py [hunk 1]
```diff
@@ -123,3 +134,27 @@ def wait_api_to_start(network: ExonumNetwork) -> None:
...
             except ConnectionError:
                 time.sleep(0.5)
+
+
+def generate_config(
+    network: ExonumNetwork,
+    deadline_height: int = 10000,
+    consensus: dict = None,
+    artifact_name: str = ARTIFACT_NAME,
+    instances: dict = {},
```
Comment (aleksuss) [inline test-suite/exonum-py-tests/suite/common.py]: This is a bad practice to use a mutable value as a default value of the argument. Take a look [here](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html).

File: test-suite/exonum-py-tests/exonum_tests/deploy.py [hunk 2]
```diff
@@ -307,32 +237,121 @@ def test_deploy_regular_stop_running_instance(self):
...
+            launcher.deploy_all()
+            launcher.wait_for_deploy()
+            launcher.start_all()
+            launcher.wait_for_start()
+
+        for validator_id in range(self.network.validators_count()):
+            host, public_port, private_port = self.network.api_address(validator_id)
+            client = ExonumClient(host, public_port, private_port)
+            available_services = client.public_api.available_services().json()
+            # crypto instance always first element in array
```
Comment (popzxc) [inline test-suite/exonum-py-tests/exonum_tests/deploy.py]: IMHO it's a bit fragile. I'd suggest to create a helper function (e.g. `find_active_service(available_service, service_name)`), which finds required service by the name and returns its status.

File: test-suite/exonum-py-tests/exonum_tests/deploy.py [hunk 3]
```diff
@@ -307,32 +238,119 @@ def test_deploy_regular_stop_running_instance(self):
...
```
Comment (popzxc) [inline test-suite/exonum-py-tests/exonum_tests/deploy.py]: Nit: This line is not required anymore.

File: test-suite/exonum-py-tests/suite/common.py [hunk 4]
```diff
@@ -123,3 +134,36 @@ def wait_api_to_start(network: ExonumNetwork) -> None:
...
+    }
+
+    return cryptocurrency_advanced_config_dict
+
+
+def find_service_status(available_service, service_name):
+    for service in available_service["services"]:
+        if service["spec"]["name"] == service_name:
+            return service["status"]["type"]
+    return "Service status has not been found"
```
Comment (popzxc) [inline test-suite/exonum-py-tests/suite/common.py]: Nit: I think it's better to raise a `RuntimeError` here.

**Pull request merged**